<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>244</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__plus_minus">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>241</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(2, 200, 255);
                         border: 1px solid rgb(0, 255, 217);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(0, 153, 255);
                         border-color: rgb(0, 153, 255);
                         }
                         QPushButton:pressed {
                         background-color: rgb(1, 139, 154);
                         border-color: rgb(1, 139, 154);
                         }</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>300</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color:rgb(255, 213, 5);
                         border: 1px solid rgb(253, 255, 135);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 193, 6);
                         border-color: rgb(255, 193, 6);
                         }
                         QPushButton:pressed {
                         background-color: rgb(193, 131, 8);
                         border-color: rgb(193, 131, 8);
                         }</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equally">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                         background-color: rgb(255, 95, 2);
                         border: 1px solid rgb(255, 143, 5);
                         color: white;
                         padding: 8px 16px;
                         
                         }
                         QPushButton:hover {
                         background-color: rgb(255, 61, 2);
                         border-color: rgb(255, 61, 2);
                         }
                         QPushButton:pressed {
                         background-color: rgb(140, 42, 3);
                         border-color: rgb(140, 42, 3);
                         }</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 251, 255);
color: rgb(42, 42, 42);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;01111111&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="input_history">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
